build-monitor-acceptance/src/test/java/features/BuildMonitorAbstractBase.java

```diff
@@ -0,0 +1,41 @@
+package features;
+
+import environment.JenkinsSandbox;
+import net.serenitybdd.integration.jenkins.JenkinsInstance;
+import net.serenitybdd.integration.jenkins.environment.rules.ApplicativeTestRule;
+import net.serenitybdd.junit.runners.SerenityParameterizedRunner;
+import net.thucydides.core.annotations.Managed;
+
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@RunWith(SerenityParameterizedRunner.class)
+public abstract class BuildMonitorAbstractBase {
```

Comment:
I wonder if we could come up with a name other than `AbstractBase` that describes the purpose of this class a bit better. Maybe `BuilMonitorAcceptanceTest`?